src/store.js

```diff
@@ -144,6 +144,19 @@ class HybridStore extends MemoryStore {
     STORE.setItem(REGISTERED_NANO_CONTRACTS_KEY, contracts)
   }
 
+  /**
+   * Unregister a nano contract.
+   *
+   * @param {string} ncId Nano Contract ID.
+   * @returns {Promise<void>}
+   * @async
+   */
+  async unregisterNanoContract(ncId) {
+    await super.unregisterNanoContract(ncId);
+    const contracts = STORE.getItem(REGISTERED_NANO_CONTRACTS_KEY) || {};
+    delete contracts[ncId];
```

Comment:
Don't you need to set the new object to `STORE`?

Reply:
No, I don't. There aren't a new object to set, `contracts` is a reference to the object, and we only need remove an entry from this object to set it right.

Reply:
Yes, I agree we should do it.

Reply:
> My suggestion is that we avoid mutating the object in this PR, use STORE.setItem to store it even though it's redundant and open a KTLO to return immutable objects in the Storage

Ok. I will open a PR. However, to achieve immutability we need more than just use the spread operator.

Reply:
I understand, we can discuss this in the KTLO, but at the very least we should return a new instance of the object, to prevent it from being mutated by the api consumer